airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -217,7 +217,7 @@ private static Container getMain(final String image,
         .withPorts(containerPorts)
         .withImage(image)
         .withImagePullPolicy(imagePullPolicy)
-        .withCommand("sh", "-c", mainCommand)
+        .withCommand("bash", "-c", mainCommand)
```

Comment:
why change to bash?

Reply:
The `>()` process substitution syntax isn't supported in `sh` https://stackoverflow.com/questions/18631499/doesnt-sh-support-process-substitution

Reply:
Got it. Can you leave a comment here?